numpy/lib/tests/test_nanfunctions.py

```diff
@@ -686,7 +688,7 @@ def test_extended_axis_invalid(self):
 
     def test_float_special(self):
         with warnings.catch_warnings(record=True):
-            warnings.simplefilter('ignore', RuntimeWarning)
+            warnings.simplefilter('always', RuntimeWarning)
```

Comment:
Just avoiding all ignore filters, catch_warnings does not propagate outside so setting it to "always" effectively ignores it. Might be a bit cleaner to ues the new context though. At the beginning I did not always.